otherlibs/stdune/test/path_tests.ml

```diff
@@ -413,3 +413,13 @@ let%expect_test _ =
   [%expect {|
 None
 |}]
+
+let%expect_test "making a directory for an existing file" =
+  let fn = Path.of_string "foo" in
```

Comment:
Can we do this in a temporary directory? Otherwise there' a chance it will interfere with other tests.

Reply:
Yes I will fix that.

Reply:
I've moved it to the more appropriate IO tests.